servicecontrol/plugins/heartbeat.md

```diff
@@ -14,6 +14,8 @@ The Heartbeat plugin enables endpoint health monitoring in ServicePulse. It send
 
 An endpoint that is marked for monitoring (by ServicePulse) will be expected to send a heartbeat message within the specified time interval. As long as a monitored endpoint sends heartbeat messages, it is marked as "active". Marking an endpoint as active means it is able to properly and periodically send messages using the endpoint-defined transport.
 
+NOTE: The Heartbeat plugin is currently not supported for Send-Only endpoints.
+
 Note that even if an endpoint is able to send heartbeat messages and it is marked as "active", other failures may occur within the endpoint and its host that may prevent it from performing as expected. For example, the endpoint may not be able to process incoming messages, or it may be able to send messages to the ServiceControl queue but not to another queue. To monitor and get alerts for such cases, develop a custom check using the CustomChecks plugin.
```

Comment:
actually there is a warning on the plugins main page stating the shutdown behavior. What if we remove that general warning and add a warning to the plugins which continue to shutdown the endpoint when SC queues are unavailable?